src/ftrackWidget.ts

```diff
@@ -278,6 +278,20 @@ export interface InitializeOptions {
   onWidgetUpdate?: (content: WidgetUpdateMessage) => void;
 }
 
+/** Get current ftrack style */
+export function getStyle() {
```

Comment:
This can be simplified using [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams#browser_compatibility)

Reply:
Code has now been updated making use of the suggested function.
Thanks !